chickadee/processes/wps_QDM.py

```diff
@@ -0,0 +1,115 @@
+from pywps import Process, LiteralInput
+from pywps.app.Common import Metadata
+from datetime import date
+from wps_tools.utils import log_handler
+from wps_tools.io import log_level, nc_output
+from chickadee.utils import logger, get_package, collect_common_args, set_r_options
+from chickadee.io import gcm_file, obs_file, varname, out_file, num_cores
+
+
+class QDM(Process):
+    def __init__(self):
+        self.status_percentage_steps = {
+            "start": 0,
+            "process": 10,
+            "build_output": 95,
+            "complete": 100,
+        }
```

Comment:
Maybe we should have considered breaking the `process step` into substeps? I feel like the processes we've implemented are not very complicated and thought it would be redundant to do so. Is there a convention for logging frequency?

Reply:
You are right we could further break it down. Really the logging is just there to provide context for what is happening behind the scenes. If you want to further break things down that works fine, otherwise we should factor out the common code.